smarts/sstudio/utils.py

```diff
@@ -28,32 +28,3 @@ class PromptFormat(Enum):
     """Default answer is yes."""
     YesNoDefaultNo = 2
     """Default answer is no."""
-
-
-def prompt_yes_no(query, format=PromptFormat.YesNoDefaultYes):
```

Comment:
I would suggest removing this entire file.

Reply:
Oh, yes, I didn't realize the `PromptFormat` is part of it.